src/playground-ide.ts

```diff
@@ -147,17 +201,64 @@ export class PlaygroundIde extends LitElement {
       >
       </playground-file-editor>
 
-      <playground-preview
-        part="preview"
-        exportparts="preview-toolbar,
-                     preview-location,
-                     preview-reload-button,
-                     preview-loading-indicator"
-        location="Result"
-        .project=${projectId}
-      ></playground-preview>
+      <div id="rhs">
+        ${this.resizable
+          ? html`<div
+              id="resizeBar"
+              @pointerdown=${this.onResizeBarPointerdown}
+            ></div>`
+          : nothing}
+
+        <playground-preview
+          part="preview"
+          exportparts="preview-toolbar,
+                       preview-location,
+                       preview-reload-button,
+                       preview-loading-indicator"
+          location="Result"
+          .project=${projectId}
+        ></playground-preview>
+      </div>
     `;
   }
+
+  async update(changedProperties: PropertyValues<this>) {
+    if (changedProperties.has('resizable') && this.resizable === false) {
+      // Note we set this property on the RHS element instead of the host so
+      // that when "resizable" is toggled, we don't reset a host value that the
+      // user might have set.
+      this._rhs?.style.removeProperty('--playground-preview-width');
+    }
+    super.update(changedProperties);
+  }
+
+  private onResizeBarPointerdown({pointerId}: PointerEvent) {
```

Comment:
I would still name private member with a `_` prefix for the benefit of JS consumers, devtools, etc.